web/src/settings_org.js

```diff
@@ -506,6 +510,16 @@ export function discard_property_element_changes(elem, for_realm_default_setting
                 property_value,
             );
             break;
+        case "realm_resolve_topic_group":
+            if (property_value !== user_groups.get_user_group_from_name("role:nobody").id) {
+                settings_components.set_dropdown_list_widget_setting_value(
+                    property_name,
+                    property_value,
+                );
+            } else {
```

Comment:
When the permission is set to `role:nobody` using checkbox, when `resolve_topic_group` is updated, we set the dropdown value with updated one only if the value !== `role:nobody`.id. Else, we just disable dropdown.